packages/site/src/components/ContentBlocks.tsx

```diff
@@ -28,7 +28,7 @@ function Block({
 }) {
   const renderers = useNodeRenderers() ?? DEFAULT_RENDERERS;
   const children = useParse(node, renderers);
-  const subGrid = 'article-grid article-subgrid-gap col-screen';
+  const subGrid = 'article-grid article-subgrid-gap col-screen mb-10';
```

Comment:
this is needed now that the top margin of the prev/next navigator has been suppressed

Reply:
I was wondering why this was cramped again. Thanks for finding and fixing it!